docs/deployment.md

```diff
@@ -119,8 +119,8 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
 1. Run `now` to deploy, replacing the `APP_ID` and `WEBHOOK_SECRET` with the values for those variables, and setting the `PRIVATE_KEY_BASE64`:
 
-        $ now -e APP_ID=aaa \
-            -e WEBHOOK_SECRET=bbb \
+        $ now -e APP_ID=1234 \
+            -e WEBHOOK_SECRET=mysecret \
             -e NODE_ENV=production \
             -e PRIVATE_KEY_BASE64="$(cat ~/Downloads/*.private-key.pem | base64)"
```

Comment:
I might have missed something here, but this isn't working for me. I don't see anywhere in Probot that we handle an environment variable named `PRIVATE_KEY_BASE64`. Was it there and I maybe had a bad merge from the TS conversion?

Reply:
I had the same thought :D https://github.com/probot/probot/pull/548/files#r191544779

It’s only used in the `now-start` script:

    "now-start": "PRIVATE_KEY=$(echo $PRIVATE_KEY_BASE64 | base64 -d) npm start"